packages/cli/src/util/env/add-env-record.ts

```diff
@@ -26,7 +28,9 @@ export default async function addEnvRecord(
     target: targets,
     gitBranch: gitBranch || undefined,
   };
-  const url = `/v8/projects/${projectId}/env`;
+  const args = upsert ? `?upsert=${upsert}` : '';
+  const version = upsert ? 'v10' : 'v8';
```

Comment:
<!-- Suggestion comments propose improvements to the code. Mention specifically what should be changed and what the benefit would be.  -->
**suggestion (blocking):** ‏use v10 always

I appreciate the careful use of v10 only for what's needed here, but I think we can and should update this call to always use v10. The only other difference between v8 and v10 is that more detailed error codes will be returned.

To accommodate, we can update the expected error codes in `util/env/known-error.ts` to be:

```
const knownErrorsCodes = new Set([
  'BAD_REQUEST',
  'ENV_ALREADY_EXISTS',
  'ENV_CONFLICT',
  'ENV_SHOULD_BE_A_SECRET',
  'EXISTING_KEY_AND_TARGET',
  'FORBIDDEN',
  'ID_NOT_FOUND',
  'INVALID_KEY',
  'INVALID_VALUE',
  'KEY_INVALID_CHARACTERS',
  'KEY_INVALID_LENGTH',
  'KEY_RESERVED',
  'RESERVED_ENV_VARIABLE',
  'MAX_ENVS_EXCEEDED',
  'MISSING_ID',
  'MISSING_KEY',
  'MISSING_TARGET',
  'MISSING_VALUE',
  'NOT_AUTHORIZED',
  'NOT_DECRYPTABLE',
  'SECRET_MISSING',
  'SYSTEM_ENV_WITH_VALUE',
  'TEAM_NOT_FOUND',
  'TOO_MANY_IDS',
  'TOO_MANY_KEYS',
  'UNKNOWN_ERROR',
  'VALUE_INVALID_LENGTH',
  'VALUE_INVALID_TYPE',
]);
```

Reply:
I can do this 🕙 

Just a query about the update of these error codes - `PAYMENT_REQUIRED` is on the current list, but not on the update you've supplied. Can you confirm that it's been removed?

Reply:
I've updated all the `url`s using the v8 env endpoint to now use v10. Also includes is the updated error code list supplied by @EndangeredMassa. See 423faf0e119361bcef0f506b3618b2fcb76a97b1

Reply:
Good eye, but yes, I left `PAYMENT_REQUIRED` off on purpose. I was comparing the list to what's used on the API side and that's not an expected code for this.